src/rgw/rgw_sal_motr.cc

```diff
@@ -4834,6 +4834,17 @@ int MotrStore::store_email_info(const DoutPrefixProvider *dpp, optional_yield y,
   return rc;
 }
 
+int MotrStore::list_gc_objs(const DoutPrefixProvider *dpp,
+              std::vector<std::unordered_map<std::string, std::string>>& gc_entries)
+{
+  auto gc = new MotrGC(cctx, this);
```

Comment:
Hi, we will try to make `motr_gc` pointer work later, but
since we are creating a temp obj with `new keyword`, let's delete it before `return statement`.

Reply:
Good catch, added deletion for gc obj.

Reply:
Great Thanks. 😊